homeassistant/components/broadlink/switch.py

```diff
@@ -238,7 +241,7 @@ def __init__(self, friendly_name, device):
         self._command_off = 0
         self._load_power = None
 
-    def _sendpacket(self, packet, retry=2):
+    def _sendpacket(self, packet, retry=CONF_RETRY):
```

Comment:
Have you tested this?
`CONF_RETRY` is a string

Reply:
I am sorry about that.  it is fixed now,  and tested pass.